src/static/css/iframe_editor.css

```diff
@@ -28,6 +28,13 @@ html.inner-editor {
   padding: 10px;
   overflow: hidden;
   background-color: white;
+
+  /* Be careful editing following rules. Longs words should not overflow, ep_align justify should work,
+     Test on chrome, firefox and safari... Copy / Paste a word inside a sentence should not add line-breaks
+     and preserve the style */
+  white-space: normal;
```

Comment:
I think this will break caret positioning.  Check the comments in the iframe_editor.css around these properties.

Reply:
Yes I read the comments about zombie caret, but actually the fix for zombie caret (white-space: pre-wrap) was already commented since a while. And I cannot reproduce the zombie caret. Or are you talking about another caret issue?